packages/flutter_tools/lib/src/commands/create.dart

```diff
@@ -142,6 +171,12 @@ class FlutterSimpleTemplate extends Template {
     // iOS files.
     files.addAll(iosTemplateFiles);
   }
+
+  void withDriverTest() {
+    additionalTemplateVariables['withDriverTest?'] = {};
+    files['test_driver/e2e.dart'] = _e2eApp;
```

Comment:
_Now_ they do. This is because we do not have integration with `@TestOn` annotations. If I put driver files in `test`, then `flutter test` will attempt to pick them up. For now, `flutter test` will operate only on `test`, and `flutter drive` will operate only on `test_driver`.